packages/react/src/primitives/types/toggleButton.ts

```diff
@@ -23,5 +23,5 @@ export interface ToggleButtonProps extends ButtonProps {
    * @description
    * Handles changes to the current value when using the ToggleButton as a controlled component
    */
-  onChange?: (value: string) => void;
+  onChange?: (value: string | undefined) => void;
```

Comment:
```suggestion
  onChange?: (value?: string) => void;
```
we shoul dbe consistent with our other components type defs; https://github.com/aws-amplify/amplify-ui/blob/main/packages/react/src/primitives/types/expander.ts#L33

Reply:
General TS guidance is to not have [optional callback parameters,](https://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html#optional-parameters-in-callbacks) so we should probably update the other component type definitions.